Pacs_Simulator/pom.xml

```diff
@@ -47,13 +47,11 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-api</artifactId>
             <version>${log4jVersion}</version>
-            <scope>provided</scope>
```

Comment:
@rohit-yawalkar @deepthi-mantena why are we packaging log4j dependencies ?

Reply:
Hi @binduak,
We are not using openmrs-api/core over here in utilities, that's why we are packaging it